src/Constants.h

```diff
@@ -0,0 +1,12 @@
+//Menu-Message-ID
+const uint32 MENU_APP_QUIT = 1000;
```

Comment:
It is not very good that these are now global. Would it be possible to keep them inside the class?
Also, since you started following Haiku coding guidelines, you could name them kMenuAppQuit, etc (the k is german for konstant or something).